src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -93,7 +93,12 @@ fi
 
 cd "$SOURCE_DIR"
 
-{{ if CustomRunBuildCommand | IsNotBlank }}
+{{ if CustomBuildCommand | IsNotBlank }}
+	echo
+	echo "Running '{{ CustomBuildCommand }}'..."
+	echo
+	{{ CustomBuildCommand }}
+{{ else if CustomRunBuildCommand | IsNotBlank }}
```

Comment:
so you are prioritizing custombuildcommand over customrunbuildcommand .... Can you site an example/use case?

Reply:
I think we need to understand it better in terms of what we are looking to do here and what npm already offers .. may be we  can leverage npm to do this what we are trying to do here.
Let's discuss this over standup ...

Reply:
https://docs.npmjs.com/cli/v6/commands/npm-ci
Currently I have two examples, one is usage of npm ci command, which allows to install dependencies in automated environment(more strict and helps catch inconsistencies). The other one is an app that containing a `Makefile`, which has its own build command, like `make build`.